ccloud/start-docker.sh

```diff
@@ -24,40 +25,59 @@ export CONFIG_FILE=~/.ccloud/config
 check_ccloud_config $CONFIG_FILE || exit 1
 check_ccloud_version 0.264.0 || exit 1
 check_ccloud_logged_in || exit 1
+printf "Done\n"
 
-#################################################################
-# Generate CCloud configurations
-#################################################################
-
+echo ====== Generate CCloud configurations
 SCHEMA_REGISTRY_CONFIG_FILE=$HOME/.ccloud/config
 ./ccloud-generate-cp-configs.sh $CONFIG_FILE $SCHEMA_REGISTRY_CONFIG_FILE
 
 DELTA_CONFIGS_DIR=delta_configs
 source $DELTA_CONFIGS_DIR/env.delta
+printf "\n"
 
-# Set Kafka cluster
+echo ====== Set Kafka cluster and service account
 ccloud_cli_set_kafka_cluster_use $CLOUD_KEY $CONFIG_FILE || exit 1
+serviceAccount=$(ccloud_cli_get_service_account $CLOUD_KEY $CONFIG_FILE) || exit 1
+ccloud kafka acl create --allow --service-account $serviceAccount --operation WRITE --topic _confluent-controlcenter --prefix
+ccloud kafka acl create --allow --service-account $serviceAccount --operation READ --topic _confluent-controlcenter --prefix
+create_connect_topics_and_acls $serviceAccount
+printf "\n"
 
-# Validate credentials to Confluent Cloud Schema Registry
+echo ====== Validate credentials to Confluent Cloud Schema Registry
 validate_confluent_cloud_schema_registry $SCHEMA_REGISTRY_BASIC_AUTH_USER_INFO $SCHEMA_REGISTRY_URL || exit 1
+printf "Done\n\n"
 
+echo ====== Starting local services in Docker
 docker-compose up -d
+printf "\n"
 
 echo "Sleeping 120 seconds to wait for all services to come up"
+printf "\n"
```

Comment:
Instead of the `sleep 120` (below), can you implement the equivalent of https://github.com/confluentinc/examples/blob/5.5.x/ccloud/start.sh#L117-L120

The main difference being the wait time and use this function instead https://github.com/confluentinc/examples/blob/5.5.x/utils/helper.sh#L708

Reply:
Nice, for sure...good eye.  WIP

Reply:
@ybyzek please see revision